kubetest/e2e.go

```diff
@@ -762,18 +762,24 @@ func kubemarkGinkgoTest(testArgs []string, dump string) error {
 }
 
 // Brings down the kubemark cluster.
-func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string) {
+func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string, logexporterGCSPath string) {
```

Comment:
Nit: You can remove "string" in "dump string" to make it: "provider, dump, logexporterGCSPath string"

In golang, if there is no type next to the parameter name, it will use the next type (so "string" in logexporterGCSPath). This pattern is already used for "provider" here which takes "string" type.